doc/assessment.md

```diff
@@ -167,3 +167,12 @@ Here is the assessment page for a closed exam with real-time grading disabled:
 ![Closed assessment with real-time grading disabled](assessment-grading-disabled-closed.png)
 
 Note that after the exam has closed and been graded, more information about points will be visible.
+
+## Honor code
+
+By default, `Exam` assessments require students to certify their identity and pledge an honor code before starting the assessment:
+
+* I certify that I am `name` and I am allowed to take this assessment in `course`.
```

Comment:
```suggestion
* I certify that I am `name` and I am allowed to take this assessment.
```

Reply:
(had already committed this change before this comment)